src/Services/testimonialService.js

```diff
@@ -1,20 +1,37 @@
 import axios from 'axios';
-
+import { AlertError } from '../Components/common/alerts/Alerts';
+import { ERROR_CREATE, ERROR_EDIT } from '../Components/common/text/textTestimonial';
 const URL = 'http://ongapi.alkemy.org/api';
 
+function verifyStatus (status,message='An error occurred') {
```

Comment:
instead of verifyStatus it should be mapError or showError, because it's not verifying any status, its showing an error